wgsl/index.bs

```diff
@@ -3808,6 +3808,109 @@ If you want `vec4<f32>` you must provide 4 float values in the constructor.
 
 Issue: (dsinclair) Write out precedence rules. Matches c and glsl rules ....
 
+Issue: (mehmetoguzderin) Refine precedence rules ....
```

Comment:
This comment and the one above can probably be removed.

Reply:
Gone now